CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+# 1.2.1
+#### Dart:
+- **feat:** additional lint rules:
+  - [unnecessary_breaks](https://dart.dev/tools/linter-rules/unnecessary_breaks)
+  - [use_rethrow_when_possible](https://dart.dev/tools/linter-rules/use_rethrow_when_possible)
+  - [use_string_in_part_of_directives](https://dart.dev/tools/linter-rules/use_string_in_part_of_directives)
+  - [constant_identifier_names](https://dart.dev/tools/linter-rules/constant_identifier_names)
+  - [use_super_parameters](https://dart.dev/tools/linter-rules/use_super_parameters)
+
+#### DCM:
+- **refactor:** remove redundant lint rule `prefer-declaring-const-constructors`
+- **refactor:** add `withBloc` getter name to `ignored-names` for `avoid-returning-widgets` in favour [prefer-getter-over-method](https://dcm.dev/docs/rules/common/prefer-getter-over-method/)
```

Comment:
This doesn't make sense since we have `withBloc` method signature that accepts parameters. Better to add exception to `prefer-getter-over-method` if possible.

Reply:
Removed for now.